flink-end-to-end-tests/test-scripts/test_streaming_file_sink.sh

```diff
@@ -22,8 +22,11 @@ OUT_TYPE="${1:-local}" # other type: s3
 source "$(dirname "$0")"/common.sh
 source "$(dirname "$0")"/common_s3.sh
 
+# randomly set up openSSL with dynamically/statically linked libraries
+OPENSSL_LINKAGE=$(if (( RANDOM % 2 )) ; then echo "dynamic"; else echo "static"; fi)
```

Comment:
add an echo message directly after the `RANDOM` what was chosen to be tested

Reply:
don't you think it is enough to print this one (during setup, in common_ssl.sh)?
```echo "Setting up SSL with: ${type} ${provider} ${provider_lib}"```

Reply:
Currently yes, but it would be safer to tie those two things together (best hidden behind some function:
```
int x = new Random().nextInt();
System.out.println(format("Executing test with following value of x = [%s] that was randomly selected", x));
```
), so that there is obvious connection between randomness and printing the random result.

Otherwise, someone might remove the logging message, without realising the implication of that fact.

Reply:
I'm not entirely convinced but also don't object and therefore added a fixup
-> I'm now waiting for the flink-shaded 7.0 release before merging